### ISSUE TITLE
Versa2 issues

### ISSUE THREAD

**Issue by hornet33** [dougwaldron/readable-fitbit-clock#8]
Hi,
I'm using Fitbit Versa 2 (Firmware 35.72.1.9) and the version of the Readable Fitbit Clock on my Fitbit is 1.1.0. 
I uninstalled and installed the clock face again today after seeing your 13/May commit comments but this is the versions (1.1.0) available to me and no, I don't get a screen with all stats after tapping on it. Maybe it's yet to be rolled out completely.
Also, another issue is that the ring around the floor counter doesn't display if it is not the primary displayed stat. My primary goal is calories, so that displays as the primary stat and the floor counter comes at the bottom row along with the rest of the non-primary stats - and that is where the ring around the floor counter doesn't display. Pic attached for reference. 
If i toggle the floor counter until it is displayed at the top (primary) display stat, then the ring comes around it - but disappears again as soon as the floor counter moves to the bottom (non-primary) display stat.
Other than these issues, this is a GREAT face and thank you for giving it to us!

![2021 05 15 - Readable Fitbit Clock - Floor Ring Issue](https://user-images.githubusercontent.com/82702970/118348240-8950d600-b566-11eb-93ed-a9bd07bcfa8b.jpg)

**Comment by dougwaldron** [comment]
Hey, thanks so much for the feedback, and I'm glad you're enjoying this watch face!

So, it takes a few days for Fitbit to approve an app or an update after it is submitted. But also, I discovered a bug in that version and had to pull it. The good news is I just submitted a new update (version 1.2.1), and it fixes the issue with the floor counter that you found. (Give Fitbit a few days to approve it.) Thanks again for reporting this!
